build/tasks/publish.js

```diff
@@ -112,16 +112,17 @@ gulp.task("npm:ui", ["npm:ui:bundles", "npm:ui:sources", "npm:ui:package", "npm:
 
 /**
  * The deliverables for clarity-angular are:
- *   - the compiled js files in es2015 format
- *   - the minified JS bundle (commonJS)
+ *   - the compiled js files in es5 with es2015 module format
+ *   - the umd JS bundle
+ *   - the minified JS bundle (in register format)
  *   - the Typescript declaration files for our components
  */
 gulp.task("npm:angular:bundles", function () {
     // We have to use 2 streams here because we want the angular 1 bundle to preserve its path.
     return es.merge(
```

Comment:
Nitpick: We don't need to merge two streams here anymore. We can just `gulp.src()` all at once.

Don't bother updating the review for this, it doesn't even matter if it stays that way, this is really a negligible nitpick.